src/components/Sidebar/drawer-list.tsx

```
@@ -0,0 +1,207 @@
+                                                            );
+                                                        },
+                                                    )}
+                                                </List>
+                                            </Collapse>
+                                        </Box>
+                                    </StyledListItem>
+                                );
+                            }
+
+                            return item.route ? (
+                                <StyledListItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)